esx_service/cli/vmdkops_admin.py

```diff
@@ -949,7 +949,7 @@ def tenant_create(args):
     """ Handle tenant create command """
     error_info, tenant = auth_api._tenant_create(name=args.name,
                                                  default_datastore=args.default_datastore,
-                                                 description="",
+                                                 description=args.description,
```

Comment:
If "args.description" is None, we may have some issue when storing it in DB. So if "args.description" is None, we should set the it to "".